addons/website_profile/data/mail_template_data.xml

```diff
@@ -4,11 +4,12 @@
 
         <!-- Email template for email validation (for karma purpose) -->
         <record id="validation_email" model="mail.template">
-            <field name="name">Profile: Email Verification</field>
+            <field name="name">Forum: Email Verification</field>
             <field name="model_id" ref="base.model_res_users"/>
             <field name="subject">{{ object.company_id.name }} Profile validation</field>
             <field name="email_from">{{ object.company_id.email_formatted or object.email_formatted }}</field>
             <field name="email_to">{{ object.email_formatted }}</field>
+            <field name="description">Sent to forum visitor to confirm their mail address.</field>
```

Comment:
```suggestion
            <field name="description">Sent to forum visitors to confirm their e-mail address</field>
```

- "address" is not necessary, could remove it
- the ending period looks unnecessary (same everywhere else in the PR)

I think Google's writing guidelines are full of good ideas on this topic:
  https://material.io/design/communication/writing.html#principles

e.g.
- write simply and directly, be concise
- skip unnecessary punctuation, specifically for solo UI sentences

@tde-banana-odoo do you know if we have internal UI guidelines that are similar to this? I couldn't find any. Perhaps we could update our coding guidelines with some UI principles.

Reply:
In this case, I'm not sure it should be considered UI since it will be only visible from the mail template tree and form view so I'm not sure the punctuation is that much out of place, though I agree some guidelines would make sense.

Reply:
We don't have internal guidelines for that but I agree with you. Sentences in description are quite long and not always to the point. I think that it is an opportunity to try to be clearer indeed.

If this is displayed in tree view, being concise is especially important ("Sent to forum visitor to confirm their mail address." -> "Ask forum visitors to confirm their email" for example).